spi/common/core-spi/src/main/java/org/eclipse/edc/spi/monitor/PrefixMonitor.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH -  initial implementation
+ *
+ */
+package org.eclipse.edc.spi.monitor;
+
+import java.util.function.Supplier;
+
+/**
+ * Monitor implementation which will prefix a supplied text to any log message.
+ * Note that this monitor will delegate the actual log execution to the underlying monitor provided by the context.
+ */
+public class PrefixMonitor implements Monitor {
```

Comment:
This can be made package private

Reply:
Are you sure? Class won't be accessible anywhere (also in the test)
 
Did you actually mean the `PrefixMonitorTest` class?